addons/stock/report/report_stockpicking_operations.xml

```diff
@@ -66,100 +50,141 @@
+                                        t-options='{"widget": "contact", "fields": ["name"], "no_marker": True}'>
+                                    </div>
+                                </div>
+                                <div name="div_state">
                                     <strong>Status:</strong>
                                     <p t-field="o.state">Draft</p>
                                 </div>
-                                <div class="col-auto" name="div_sched_date">
-                                    <strong>Scheduled Date:</strong>
-                                    <p t-field="o.scheduled_date">2023-09-24</p>
+                                <div t-if="o.user_id" name="div_operator">
+                                    <strong>Operator:</strong>
+                                    <p t-field="o.user_id">Mitchel Admin</p>
+                                </div>
+                                <div name="div_sched_date">
+                                    <strong t-esc="'Done Date:' if o.state == 'done' else 'Scheduled Date:'">Scheduled Date:</strong>
+                                    <p t-field="o.scheduled_date" t-options='{"widget": "date"}'>2023-09-24</p>
+                                </div>
+                                <div name="div_vendor" t-if="o.picking_type_id.code=='incoming' and o.partner_id">
+                                    <strong>Vendor:</strong>
+                                    <div t-field="o.partner_id.commercial_partner_id"
+                                        t-options='{"widget": "contact", "fields": ["name", "phone"], "no_marker": True, "phone_icons": True}'>
+                                    </div>
                                 </div>
                             </div>
-                            <table class="table table-sm" t-if="o.move_line_ids and o.move_ids_without_package">
-                                <t t-set="has_barcode" t-value="any(move_line.product_id and move_line.product_id.sudo().barcode or move_line.package_id for move_line in o.move_line_ids)"/>
-                                <t t-set="has_serial_number" t-value="any(move_line.lot_id or move_line.lot_name for move_line in o.move_line_ids)" groups="stock.group_production_lot"/>
+                            <t t-set="format_number" t-value="lambda x: int(x) if x.is_integer() else '{:.2f}'.format(x)"/>
+                            <table class="table table-sm" t-if="o.move_line_ids and o.move_ids_without_package" style="table-layout: fixed; width: 100%">
+                                <t t-set="picking_has_barcode" t-value="any(move_line.product_id and move_line.product_id.sudo().barcode or move_line.package_id for move_line in o.move_line_ids)"/>
+                                <t t-set="picking_has_serial_number" t-value="any(move_line.lot_id or move_line.lot_name for move_line in o.move_line_ids)" groups="stock.group_production_lot"/>
```

Comment:
They serve different purposes. The first one to indicate if we should add the barcode column, the other is needed to indicate if we want to do the list breakdown.

Reply:
Mm, the list breakdown needs to happen for non-tracked products as well. I think I didn't make it clear with my other comments about the locations/packages.

For example is this use case:
![image](https://github.com/odoo/odoo/assets/57942143/49334bc7-7cdc-4d39-ade6-bdaa61c7072f)

the picking operations current print as this (in master):
![image](https://github.com/odoo/odoo/assets/57942143/881d287e-b09b-476e-ad4a-20727653dfa2)

whereas your changes make it print like this (I did this in your PR's runbot, so you can see it there):
![image](https://github.com/odoo/odoo/assets/57942143/81b842d8-e1b2-4d64-8266-f5cd5000da11)


We shouldn't lose this granularity. I.e. instead of checking for `picking_has_serial_number` to see if the list breakdown should occur, you should check `len(o.move_line_ids) > 1`, which would be checked per move rather than per picking